torch/_inductor/ir.py

```diff
@@ -4011,7 +4011,31 @@ def convert_to_reinterpret_view(cls, x):
 
         # NOTE: Don't use extract_read_writes here as it fails when
         # make_loader() inlines the computation
-        x.unwrap_view().freeze_layout()
+        x_unwrap_view = x.unwrap_view()
+        x_unwrap_view_node = V.graph.get_buffer(
+            x_unwrap_view.get_name()
+        ).get_origin_node()
```

Comment:
`get_origin_node` may not always work as comment in https://github.com/pytorch/pytorch/blob/1fd9e320ea415c4b221799863a334c842bdb6ff2/torch/_inductor/graph.py#L1254-L1261 But here since we check and decide the CL format only when `get_origin_node` exists. I think it should be ok.